packages/swapper/src/swappers/zrx/ZrxSwapper.test.ts

```diff
@@ -60,7 +60,7 @@ describe('ZrxSwapper', () => {
     const { buildTradeInput } = setupBuildTrade()
     const swapper = new ZrxSwapper(zrxSwapperDeps)
     const args = { ...buildTradeInput, wallet }
-    await swapper.buildTrade(args)
+    await swapper.buildTrade({ ...args, chainId: 'eip155:1' })
```

Comment:
should just add the chainId to the `setupBuildTrade` mock. also do we want to remove wallet from the eth ones knowing that isn't used/required? guess can wait until the next PR to remove that from the common type as well.

Reply:
it doesnt work with types like that for some reason